### DISCUSSION TITLE
steam 冬季特賣

### DISCUSSION THREAD

**Discussion by wangtyphoon** [1101-GS4719-C/Python-Community-GS4719-C#979] [General]
這個冬季特賣我買了巫師三和戰地風雲五，實在是太便宜了，剩最後半天，走過路過不要錯過，最近這幾年遊戲的畫面隨著顯卡性能的提升，各項特效都有顯著的進步，我最喜歡monster hunter world啦!
但除此之外我們來談談nvidia獨步武林的絕佳技術dlss，DLSS 是人工智慧繪圖技術的革命性突破，可大幅提升效能。DLSS 3 採用 GeForce RTX 40 系列 GPU 上的全新第四代 Tensor 核心和光流加速器，可運用人工智慧創造額外的高品質畫面，能夠有效提升遊戲時的禎數表現
![image](https://user-images.githubusercontent.com/109240416/210718115-9eb9aa74-74e6-4461-b912-f33696b1655a.png)

**Comment by wanderer657** [comment]
這次特賣給了玩家絕佳機會，購得《巫師三》和《戰地風雲五》等佳作。隨著科技進步，遊戲畫面和特效提升明顯，增強了沉浸感。我最愛《怪物獵人世界》。DLSS技術引人注目，利用人工智慧提升畫面效能，是遊戲界的革命。DLSS 3搭載全新Tensor核心和光流加速器，提升畫面品質和流暢度。與其他技術相比，DLSS明顯更優越，為遊戲帶來全新體驗。

**Comment by thomasxshen** [comment]
巫師三推推，我整個寒假都在玩，巫師三不僅常常打折，DLC血與酒和石之心還免費送，CDPR真的是業界良心！而且在冬季特賣時，巫師三還迎來了次世代升級，畫面效能全面提升，還能支持光追，完全看不出來是2015年的遊戲。只可惜，我的GeForce RTX 3050實在無法承受DLSS技術，只能把效能調到最低，才能體驗巫師三所架構的龐大世界。
